Src/ILGPU/Runtime/CPU/CPUAccelerator.cs

```diff
@@ -108,7 +108,8 @@ public sealed partial class CPUAccelerator : Accelerator
                     UsesSequentialExecution);
             }
 
-            Bind();
+            if (!description.Equals(CPUDevice.Implicit))
```

Comment:
I know that the solution feels kind of hacky... However, I think we can get rid of the call to `Bind` anyway... right?

Reply:
This call to `Bind` is the one that initializes `Accelerator.Current` when the user calls `context.CreateCPUAccelerator`. There are similar calls in the constructor of the other accelerators.

We could potentially move it out of the constructor, and into the `CreateCPUAccelerator`, `CreateCudaAccelerator` and `CreateCLAccelerator` methods?

Reply:
OK 👍